modopt/opt/proximity.py

```diff
@@ -688,3 +689,353 @@ def _cost_method(self, *args, **kwargs):
             print(' - ELASTIC NET (X):', cost_val)
 
         return cost_val
+
+
+class KSupportNorm(ProximityParent):
+    r"""K-support norm proximity operator
+
+    This class defines the squarred K-support norm proximity operator
+    described in [M2016].
+
+    \underset{y \in \mathbb{C}^{N}}{\text{min}} 0.5*\|x - y|\_2^2 +
+    \frac{\beta}{2}\text{min}\{\sum_{I \in \mathcal{G}_k}\|v_I\|_2^2:
+    \text{supp} (v_I) \subseteq I, \sum_{I \in \mathcal{G}_k} v_I = y\}
```

Comment:
I don't think this will render without the `.. math::` command.

Reply:
The expression is a bit complex, maybe we should just refer the user to the papers in references?

Reply:
That would be completely acceptable

Reply:
I think I will refer to the paper